### PR TITLE
ADEN-2249 Clean up the ads-monitoring-oasis tests

### PR DESCRIPTION
**Pull request by gabrys**

### CODE REVIEW

File: src/test/java/com/wikia/webdriver/common/core/configuration/Configuration.java [hunk 1]
```diff
@@ -136,4 +136,8 @@ public static void setTestValue(String key, String value) {
   public static void clearCustomTestProperties() {
     testConfig.clear();
   }
+
+  public static String getCountryCode() {
```
Comment (ludwikkazmierczak) [inline src/test/java/com/wikia/webdriver/common/core/configuration/Configuration.java]: Add a default country code to sample_config

File: src/test/java/com/wikia/webdriver/common/core/geoedge/GeoEdgeBrowserMobProxy.java [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+/**
+ * @author Bogna 'bognix' Knychala
+ */
+@Retention(value = RetentionPolicy.RUNTIME)
+@Target(value = {ElementType.METHOD, ElementType.TYPE, ElementType.CONSTRUCTOR})
+public @interface GeoEdgeBrowserMobProxy {
```
Comment (ludwikkazmierczak) [inline src/test/java/com/wikia/webdriver/common/core/geoedge/GeoEdgeBrowserMobProxy.java]: a CountryProxy name would be just fine

File: src/test/java/com/wikia/webdriver/common/core/configuration/Configuration.java (hunk 1 above)
Comment (gabrys) [inline src/test/java/com/wikia/webdriver/common/core/configuration/Configuration.java]: It needs to be null by default (no proxy please)

File: src/test/java/com/wikia/webdriver/common/core/geoedge/GeoEdgeBrowserMobProxy.java (hunk 2 above)
Comment (gabrys) [inline src/test/java/com/wikia/webdriver/common/core/geoedge/GeoEdgeBrowserMobProxy.java]: I want to get rid of this old class eventually, so the name doesn't matter. I'm just updating Oasis for now, Mercury and WikiaMobile are planned next.

**Comment by drets** [comment]
LGTM

**Pull request merged**